src/Wallabag/CoreBundle/Command/ExportCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Wallabag\CoreBundle\Command;
+
+use Doctrine\ORM\NoResultException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\StreamOutput;
+
+class ExportCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wallabag:export')
+            ->setDescription('Export all entries for an user')
+            ->setHelp('This command helps you to export all entries for an user')
+            ->addArgument(
+                'username',
+                InputArgument::REQUIRED,
+                'User from which to export entries'
+            )
+            ->addArgument(
+                'filepath',
+                InputArgument::OPTIONAL,
+                'Path of the exported file'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $user = $this->getUser($input->getArgument('username'));
+        } catch (NoResultException $e) {
+            $output->writeln(sprintf('<error>User "%s" not found.</error>', $input->getArgument('username')));
+            return 1;
+        }
+        $entries = $this->getDoctrine()
+            ->getRepository('WallabagCoreBundle:Entry')
+            ->getBuilderForAllByUser($user->getId())
+            ->getQuery()
+            ->getResult();
+
+        $output->write(sprintf('Exporting %d entrie(s) for user « <comment>%s</comment> »... ', count($entries), $user->getUserName()));
+
+        $filePath = $input->getArgument('filepath');
+        if (!$filePath) {
+            $filePath = $this->getContainer()->getParameter('kernel.root_dir') . '/../' . sprintf('%s-export', $user->getUsername());
+        }
```

Comment:
I don't if it's a good thing to have export filenames based on username (even if I know that these files are not available from the web server).